AdminUi/src/AdminUi/ClientApp/.ci/runChecks.sh

```diff
@@ -0,0 +1,9 @@
+set -e
+set -x
+
+npm ci
+npx eslint --ext ts ./src
+npx prettier --check .
+npx license-check --ignorePackages adminui@0.0.0
```

Comment:
You should be able to skip the version

```suggestion
npx license-check --ignorePackages adminui
```

Reply:
I tried it without the version, but it isn't ignoring the package without the version.